core/services/ocr2/plugins/functions/plugin.go

```diff
@@ -106,7 +108,26 @@ func NewFunctionsServices(ctx context.Context, functionsOracleArgs, thresholdOra
 
 	offchainTransmitter := functions.NewOffchainTransmitter(DefaultOffchainTransmitterChannelSize)
 	listenerLogger := conf.Logger.Named("FunctionsListener")
-	bridgeAccessor := functions.NewBridgeAccessor(conf.BridgeORM, FunctionsBridgeName, MaxAdapterResponseBytes)
+
+	var maxRetry int
+	if pluginConfig.ExternalAdapterMaxRetry != nil {
+		maxRetry = int(*pluginConfig.ExternalAdapterMaxRetry)
+		conf.Logger.Debugf("external adapter maxRetry configured to: %d", maxRetry)
+	} else {
+		maxRetry = DefaultMaxAdapterRetry
+		conf.Logger.Debugf("external adapter maxRetry configured to default: %d", maxRetry)
```

Comment:
you can just have a single log message below

Reply:
👍🏼 true!  I wanted to make a clear difference between picking up the value from the jobspec vs using the default value, but if we are logging the actual value it should be enought!